helion/_compiler/device_ir.py

```diff
@@ -129,9 +129,9 @@ def _get_proxy_slot(
         ),
     ):
         current_location().set_fx_location()
-        return proxy_tensor.make_fx(fn, decomposition_table=select_decomp_table())(
-            *args
-        ).graph
+        decomp_table = select_decomp_table()
+        decomp_table.pop(torch.ops.aten.stack.default, None)
```

Comment:
select_decomp_table is cached, so there is a *single* dict here that you are mutating.

This means that you are modifying the behavior of torch.compile since it will use the modified table.  Make a copy.